apis/identity/v1beta1/iamrole_types.go

```diff
@@ -14,15 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha3
+package v1beta1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/aws/aws-sdk-go-v2/service/iam"
-
-	aws "github.com/crossplane/stack-aws/pkg/clients"
-
 	runtimev1alpha1 "github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
 )
 
```

Comment:
It looks like there are some additional configurable fields for `IAMRole` that can be exposed: https://github.com/aws/aws-sdk-go-v2/blob/9fc62ee75d1acca973ac777e51993fce74f6a27f/service/iam/api_op_CreateRole.go#L13